indico/htdocs/sass/modules/_roombooking.scss

```diff
@@ -150,10 +150,37 @@
         @include border-radius();
         width: 370px;
     }
+
+    #booked-for-user-wrapper {
+        width: 240px;
+        display: none;
+        min-height: 0;
+
+        &.hasError {
+            @include border-all($red);
+        }
+    }
+
+    #other-user {
+        display: table-cell;
```

Comment:
Not very important, but did you check whether you can do the same with flexbox? :smile: If it takes too long and the layout is already implemented with tables, don't waste too much time, but I'd prefer seeing flexbox attributes in newly added code! :laughing: